### ISSUE TITLE
eChecklists not exporting properly when multiple ST&Es exist. 

### ISSUE THREAD

**Issue by mshuter** [cyberperspectives/sagacity#96]
After importing and categorizing the same data into two different ST&Es, one of the ST&Es no longer exports eChecklists properly - it creates the file but it has no eChecklist tabs, only Cover Sheet and Instructions.  

I was able to replicate this issue using the sample eChecklist in the shared folder named "Win_10_-_x64-100hosts-modified-eChecklist-29Nov18.xlsx" (I trimmed off hosts after JOAB-67, not sure if it would work with less).  

Steps to reproduce:

1. Create two Systems, two Sites, and two ST&Es.  The data I used was Test System, Test Site, and ST&E 1 (the pre-loaded test data) and Test System 2, Test Site 2, and ST&E 2. 
2. Import trimmed checklist into first ST&E. Auto-categorize. Verify export works correctly. 
3. Import trimmed checklist into second ST&E. Auto-categorize.  Click export button.  File exports but checklist tabs are not present.

**Comment by mshuter** [comment]
Digging a little it looks like get_cat_findings only has hosts in it with tgt_id < 66 (i.e., none of the hosts from the second import).  This probably explains the symptom... are you matching findings on import based only on hostname and not also ST&E ID?

**Comment by godsgood33** [comment]
> Digging a little it looks like get_cat_findings only has hosts in it with tgt_id < 66 (i.e., none of the hosts from the second import). This probably explains the symptom... are you matching findings on import based only on hostname and not also ST&E ID?

Targets have to be linked directly to an ST&E ID, so they are unique as necessary.

Were you able to figure anything else out about this?

**Comment by mshuter** [comment]
No, just like I said, you can replicate it by importing the same unclass echecklist with ~67 hosts into two separate ST&Es.

________________________________
